mne/epochs.py

```diff
@@ -508,6 +508,42 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
                            scalings=scalings, title_str=title_str,
                            show=show, block=block)
 
+    def plot_concat(self, picks=None, scalings=None, n_epochs=8,
+                    n_channels=10, title_str='#%003i', show=True, block=False):
```

Comment:
There are also n_channels, n_epochs and bad_color. I'm not sure we need the bad_color to be changeable though.